src/NuGet.Clients/NuGet.Console/NuGet.Console.csproj

```diff
@@ -9,6 +9,7 @@
     <TargetFramework>$(NETFXTargetFramework)</TargetFramework>
     <UseWpf>true</UseWpf>
     <ExcludeFromSourceOnlyBuild>true</ExcludeFromSourceOnlyBuild>
+    <NoWarn>$(NoWarn);CS1572;CS1573;CS1591;IDE0005;IDE1006</NoWarn>
```

Comment:
The PR title says it's enforcing IDE0005, but it's just being suppressed here, not enforced.